addons/api/addon/models/worker.js

```diff
@@ -1,27 +1,35 @@
 import GeneratedWorkerModel from '../generated/models/worker';
-import { fragment } from 'ember-data-model-fragments/attributes';
+import { attr } from '@ember-data/model';
 
 export default class WorkerModel extends GeneratedWorkerModel {
   // =attributes
 
-  /**
-   * @type {[FragmentTagModel]}
-   */
-  @fragment('fragment-tag', {
+  @attr({
     description:
       'The deduplicated union of the tags reported by the worker ' +
       'from its configuration and any tags added through other means.\nOutput only.',
     readOnly: true,
   })
   canonical_tags;
 
-  /**
-   * @type {[FragmentTagModel]}
-   */
-  @fragment('fragment-tag', {
+  @attr({
     description:
       "The tags set in the worker's configuration file.\nOutput only.",
     readOnly: true,
   })
   config_tags;
+
+  addWorker(workerGeneratedAuthToken, options = { adapterOptions: {} }) {
+    const defaultAdapterOptions = {
+      method: 'add-worker',
```

Comment:
Is the method `create:worker-led`?

Reply:
no that was the initial pass of me adding it.  Should have waited to commit and push when all of the changes were done